image/task.go

```diff
@@ -0,0 +1,31 @@
+package image
+
+// Task is an image processing task
+type Task struct {
+	Buffer         []byte
+	Width          int
+	Height         int
+	BlurAmount     int
```

Comment:
Are you sure you don't want a boolean for whether or not to perform a blur? You're storing two kinds of information in this variable otherwise, whether or not to perform a blur and how much to blur, you can avoid blur logic altogether with a boolean ApplyBlur variable if you wanted to.

Reply:
Absolutely do want a boolean for that, done